src/routes/sessionReports/handlers.js

```diff
@@ -166,3 +172,23 @@ export const getParticipants = async (req, res) => {
     return handleErrors(req, res, error, logContext);
   }
 };
+
+export async function getGroups(req, res) {
+  const { region } = req.query;
+  const userId = await currentUserId(req, res);
+  const user = await userById(userId);
+  const regionNumber = parseInt(region, DECIMAL_BASE);
+  const eventPolicy = new EventReport(user, { regionNumber });
+  // Has correct TR permissions.
+  if (!eventPolicy.canWriteInRegion(regionNumber)) {
```

Comment:
A different end point was added calling the same service function as the permission to check are diff AR vs TR.